package.json

```diff
@@ -62,6 +62,11 @@
     "type": "git",
     "url": "https://github.com/angularclass/NG6-starter.git"
   },
+  "babel": {
```

Comment:
Ideally, it should be enough to have it in `.babelrc`. Unfortunately, that didn't work for me. May be that is because I don't have `babel` installed globally.

Wouldn't be better if we have it at both places to ensure the build doesn't fail?